ldk/javascript/src/whisper/mapper.ts

```diff
@@ -379,6 +379,8 @@ export function mapToInternalChildComponent(
           component.onChange(error, param, mapToExternalWhisper(whisper, stateMap));
         },
       } as WhisperService.TextInput;
+    case WhisperComponentType.Typography:
```

Comment:
same as above alphabetical comment

Reply:
moved dateTimeInput and Icon to correct spot